modules/ROOT/pages/changelog.adoc

```diff
@@ -5,6 +5,16 @@
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
 
+== 6.7.2 - 2023-10-25
+
+=== Fixed
+* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
+* The function `getModifierState` did not work on events passed through the editor as expected.
+* List items containing a nested list between two text caused some list operations to fail.
+* Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.
+* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.
+* If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.
```

Comment:
@ArvinJ-H Can you confirm that is has been moved to 6.8, so i can remove from 6.7.2 community release.
```suggestion
```